src/stats_hero_sup.erl

```diff
@@ -34,6 +34,7 @@
 -define(SERVER, ?MODULE).
 
 start_link() ->
+    {ok, _} = pg:start_link(),
```

Comment:
This will attach pg to this supervisor, but not make it part of the supervision tree -- iirc, a crash of `pg` will cause the supervisor itself to get restarted in that case? 

Could we instead set up a `new_child` function that creates the right record for supervising the process, passing in `pg`, `start_link` for MF args, so that it's supervised as the other child procs are?

Reply:
So, after very much wailing and gnashing of teeth trying to get a supervision tree output (VERY DIFFICULT), testing the old behavior (impossible at least for me - some kind of error with observer), and testing this new behavior above, it seems crashing pg doesn't cause the supervisor itself to crash from what I can see, but it DOES result in not restarting pg.

I'll follow your advice above and see what I can come up with.

Reply:
![image](https://user-images.githubusercontent.com/51833247/178076118-1b37bac3-4ff6-4640-a10d-9e4e22bee988.png)
This depicts the current supervision tree.  Here, pg will not restart if I crash it.

Reply:
![image](https://user-images.githubusercontent.com/51833247/178079431-3473fe64-c0f7-4f44-a112-d239da2c8e87.png)
Revamped supervision tree (pg moved under stats_hero_sup).  pg confirmed to restart upon crash.

Reply:
I did note that if `pg` was started first by `stats_hero_sup` and `new_child` and was first in the `CHILD_TYPES` list, the eunit tests crashed and things weren't set up properly.  I did not have time to investigate further, and things seem to work when `pg` is last, but if anything pops into your head regarding this, let me know.

Reply:
```
$ ./rebar3 eunit
<snip>
Finished in 0.969 seconds
29 tests, 0 failures
```